libraries/chain/include/eosio/chain/block_state.hpp

```diff
@@ -127,7 +127,11 @@ struct block_state : public block_header_state {     // block_header_state provi
       return timestamp() > fc::time_point::now() - fc::seconds(30);
    }
 
-   block_ref make_block_ref() const { return block_ref{block_id, timestamp(), strong_digest }; } // use the cached `finality_digest`
+   // use the cached `finality_digest`
+   block_ref make_block_ref() const {
```

Comment:
Why do we define a new `make_block_ref` here? We already have one in block_header_state.